sdk/communication/azure-communication-identity/swagger/SWAGGER.md

```diff
@@ -35,4 +36,14 @@ directive:
   - from: swagger-document
     where: $.definitions.CommunicationIdentityTokenScope
     transform: >
-      $["x-ms-enum"].name = "CommunicationTokenScope";
\ No newline at end of file
+      $["x-ms-enum"].name = "CommunicationTokenScope";
+```      
+
+### Removes the "format": "date-time" from property expiresOn
+```yaml
```

Comment:
can you pls add a comment about why we introduced this transformation for future reference